### PR TITLE
Bump version to 1.0.2 and backport postgres fix

### PR DESCRIPTION
**Pull request by JackMc**

### CODE REVIEW

**Comment by JackMc** [comment]
@wardi Please CR - blocking deploy

**Comment by wardi** [comment]
It's not a backport if it's not on master :-) @JackMc I'm waiting for your test on https://github.com/datacats/datacats/pull/249 then I'll put out 1.0.2

**Comment by JackMc** [comment]
Aww ok will close

**Comment by wardi** [comment]
Please add an entry to docs/changelog for this release

**Comment by JackMc** [comment]
Is that good? :)

**Comment by JackMc** [comment]
My technical-writing-foo is meh

File: docs/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Change Log
 ==========
 
+1.0.2
+-----
+2015-07-07
+
+- Compatibility with Docker 1.7.0 by removing reliance on ENTRYPOINT.
+- Fix an issue with datapusher where the site_url would not be set 
+  correctly for CKAN to respond.
```
Comment (wardi) [inline docs/changelog.rst]: this particular change doesn't seem to be on the branch. Let me find it and cherry-pick it as well

**Comment by wardi** [comment]
pushed these and the missing commit to the latest-1.0 branch

**Pull request closed without merging**